update_automation_example.py

```diff
@@ -0,0 +1,124 @@
+        else:
+            options.name = os.path.splitext(
+                os.path.basename(options.file.name))[0]
+    return options
+
+
+class TestAutomation(unittest.TestCase):
+    def setUp(self):
+        automation = {
+            "name": self.name,
+            "script": "demisto.results('Hello, world!')"
+        }
+        automations = self.client.SaveAutomation(automation)
+        self.id = automations["scripts"][0]["id"]
+
+    def tearDown(self):
+        automation = {
+            "name": self.name,
+            "id": self.id
+        }
+        self.client.DeleteAutomation(automation)
+        self.id = None
+
+    def test_search(self):
+        automations = self.client.SearchAutomation("name:{}".format(self.name))
+        self.assertTrue("scripts" in automations)
+        self.assertEqual(len(automations["scripts"]), 1)
+        self.assertTrue("id" in automations["scripts"][0])
+
+    def test_delete(self):
+        automation = {
+            "name": self.name,
+            "id": self.id
+        }
+        client.DeleteAutomation(automation)
+        automations = self.client.SearchAutomation("name:{}".format(self.name))
+        self.assertTrue("scripts" in automations)
+        self.assertEqual(len(automations["scripts"]), 0)
+
+    def test_save_new(self):
+        automation = {
+            "name": self.name,
+            "id": self.id
+        }
+        client.DeleteAutomation(automation)
+        automation = {
+            "name": self.name,
+            "script": "demisto.results('Hello, world!')"
+        }
+        automations = self.client.SaveAutomation(automation)
+        self.assertTrue("scripts" in automations)
+        self.assertEqual(len(automations["scripts"]), 1)
+        self.assertTrue("id" in automations["scripts"][0])
+        self.id = automations["scripts"][0]["id"]
+
+    def test_load(self):
+        automation = self.client.LoadAutomation(self.id)
+        self.assertTrue("id" in automation)
+        self.assertEqual(automation["id"], self.id)
+
+    def test_update(self):
+        script = "demisto.results('Hello, world!!!!')"
+        automations = self.client.UpdateAutomation(
+            None, self.name, script=script)
+        self.assertTrue("scripts" in automations)
+        self.assertEqual(len(automations["scripts"]), 1)
+        self.assertTrue("version" in automations["scripts"][0])
+        self.assertEqual(automations["scripts"][0]["version"], 2)
+
+
+if __name__ == '__main__':
+    options = options_handler()
+    client = demisto.DemistoClient(options.key, options.url)
+    if options.test:
+        TestAutomation.name = options.name
+        TestAutomation.client = client
+        if sys.argv:
+            del sys.argv[1:]
+        unittest.main()
+        sys.exit(0)
+    script = options.file.read()
+    automations = client.UpdateAutomation(None, options.name, script=script)
+    print(json.dumps(automations))
+    sys.exit(0)
```

Comment:
What's the purpose of using `sys.exit(0)` here? Isn't the script already exiting?

Reply:
unnecessary. removed.(304fa52)